mllib/src/main/scala/org/apache/spark/ml/classification/LogisticRegression.scala

```diff
@@ -295,6 +295,13 @@ class LogisticRegression @Since("1.2.0") (
     instr.logParams(regParam, elasticNetParam, standardization, threshold,
       maxIter, tol, fitIntercept)
 
+    val autoAggregationDepth =
+      if ($(aggregationDepth) > 0) $(aggregationDepth)
+      else getAggregationDepthByFormula(
+        instances.context.getConf.getSizeAsBytes("spark.driver.memory", "1g"),
```

Comment:
oh, that is because, when user not set "spark.driver.memory", spark will set it as "1g" automatically,
but in such case `context.getConf.getSizeAsBytes("spark.driver.memory")` will throw NoSuchElement exception. so I need to get the spark internal default value `1g`, is there a better way?